MapboxNavigation/RouteVoiceController.swift

```diff
@@ -170,19 +175,23 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
         // Set recentlyAnnouncedRouteStep to the current step
         recentlyAnnouncedRouteStep = routeProgress.currentLegProgress.currentStep
         
-        fallbackText = speechString(notification: notification, markUpWithSSML: false)
+        fallbackText = createSpeakText(from: routeProgress, at: userDistance)
         
         // If the user is merging onto a highway, an announcement to merge is a bit excessive
-        if let upComingStep = routeProgress.currentLegProgress.upComingStep, routeProgress.currentLegProgress.currentStep.maneuverType == .takeOnRamp && upComingStep.maneuverType == .merge && routeProgress.currentLegProgress.alertUserLevel == .high {
+        if let upComingStep = routeProgress.currentLegProgress.upComingStep,
+            routeProgress.currentLegProgress.currentStep.maneuverType == .takeOnRamp,
+            upComingStep.maneuverType == .merge,
+            routeProgress.currentLegProgress.alertUserLevel == .high {
             return false
         }
         
         return true
     }
     
-    func speechString(notification: NSNotification, markUpWithSSML: Bool) -> String {
-        let routeProgress = notification.userInfo![RouteControllerAlertLevelDidChangeNotificationRouteProgressKey] as! RouteProgress
-        let userDistance = notification.userInfo![RouteControllerAlertLevelDidChangeNotificationDistanceToEndOfManeuverKey] as! CLLocationDistance
+    /**
+     Creates an audio instruction from a `RouteProgress`, the users distance from the maneuver. Optional `markUpWithSSML` will apply `SSML` to the string. This is useful when using AWS's Polly speech synthesizer.
+    */
+    public func createSpeakText(from routeProgress: RouteProgress, at userDistance: CLLocationDistance, markUpWithSSML: Bool = false) -> String {
```

Comment:
`createSpeakText` is ungrammatical; a better method name would be `speechString(describing:at:markUpWithSSML:)`.